modules/web-console/e2e/testcafe/mocks/agentTasks.js

```diff
@@ -211,3 +211,17 @@ export const agentStat = (clusters) => (ws) => {
 export const errorResponseForEventType = (eventType) => (ws) => {
     ws.errorOn(eventType, () => DFLT_FAILURE_RESPONSE);
 };
+
+export const TEST_JDBC_IMPORT_DATA = {
+    drivers: [{
+        'jdbcDriverJar': 'test-driver.jar',
+        'jdbcDriverClass': 'test.dvired.cls',
```

Comment:
Is that **dvired** correct?
Or it is not important?

Reply:
It is not important. Test of the import require a real configured database or should be fully mocked.